src/platforms/platforms.tsx

```diff
@@ -103,6 +107,12 @@ type PlatformData<
   }
 >
 
+const AddPlatformPanelLoadingSpinner = () => (
+  <Center w="full" h="51vh">
+    <Spinner />
```

Comment:
I think the spinner could be bigger, but otherwise it looks good to me!

Reply:
okay, I'll fix it

Reply:
<img width="888" alt="image" src="https://github.com/guildxyz/guild.xyz/assets/66187782/7bde9506-a6e3-415c-85e6-648a20b9da59">
XL maybe better than L because the content size quite big.